Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -221,6 +221,70 @@ INT_VECT EmbedMultipleConfs(
     bool enforceChirality = true, bool useExpTorsionAnglePrefs = false,
     bool useBasicKnowledge = false, bool verbose = false,
     double basinThresh = 5.0);
+
+struct EmbedParameters {
+  unsigned int maxIterations;
```

Comment:
It looks like there was an issue with vector based setters/getters that don't expose the proxy class that bit me.

http://stackoverflow.com/questions/35203463/swig-getters-setters-for-an-array-of-struct-dont-return-expect-an-array-of-pro

This looks good then.